src/patch.jl

```diff
@@ -0,0 +1,25 @@
+using GeoInterface
```

Comment:
@SimonDanisch should this file be renamed something like `geoJSONtraitParse` if we're going to keep it in GeoMakie?